src/main/java/com/novauc/SpringBasicsFrApplication.java

```diff
@@ -2,11 +2,64 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+
+import javax.servlet.http.HttpSession;
+import java.util.ArrayList;
 
 @SpringBootApplication
+@Controller
 public class SpringBasicsFrApplication {
 
+public static ArrayList<Message> themessages = new ArrayList<>();
+
+	@RequestMapping(path = "/", method = RequestMethod.GET)
```

Comment:
Generally, you would not mark you application class as a controller.  You normally want all your controller stuff to be in it's own class.  No big deal though.  This works really well, good job Ross